### ISSUE TITLE
FigureResampler not working when using backend parameter on ServersideOutputTransform

### ISSUE THREAD

**Issue by VictorBauler** [predict-idlab/plotly-resampler#132]
Hello and thank you for this awesome project.

I'm trying to specify backend parameter on ServersideOutputTransform in order to set threshold parameter on FileSystemCache, but then if I do that my callback with fig.construct_update_data(relayoutdata) stop working. Any idea about why this is happening?

Code:
```` python
from dash import html, dcc, Output, Input, ctx, State, no_update
import dash_bootstrap_components as dbc
import plotly.graph_objects as go

from dash_extensions.enrich import (
    DashProxy,
    ServersideOutput,
    ServersideOutputTransform
)

from plotly_resampler import FigureResampler
from trace_updater import TraceUpdater
from flask_caching.backends import FileSystemCache

from settings import DROPDOWN_OPTIONS
from preprocess import Preprocess
# ----------------- Defining Global Variable with all data ---------------------

data = Preprocess()
data.preprocess_all_data()


def get_plot(plot_domain, acc, file):
    return data.get_plot(plot_domain, acc, file)


def get_stats(plot_domain, acc, file):
    return data.get_stats(plot_domain, acc, file)

# ---------------------- Create App ----------------------------

backend = FileSystemCache(cache_dir='file_system_store',
                           threshold=3)


app = DashProxy(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP],
                transforms=[ServersideOutputTransform(backend=backend)])


def create_figure(files_list=['a_normal__2000rpm'], plot_domain='time', acc=0):

    fig = FigureResampler(go.Figure(), default_n_shown_samples=10_000)

    if plot_domain == 'fft':

        for file in files_list:

            fig.add_trace(get_plot(plot_domain, acc, file))

        fig.update_layout(
            yaxis_title='Amplitude',
            xaxis_title='Frequency (Hz)',
            width=1400,
            height=600)

    elif plot_domain == 'time':

        for file in files_list:

            fig.add_trace(get_plot(plot_domain, acc, file))

        fig.update_layout(
            yaxis_title='Amplitude (g)',
            xaxis_title='Time (s)',
            width=1400,
            height=600)

    return fig


app.layout = html.Div(
    [
        dbc.Row(
            dbc.Col([
                html.H1('Medições de Bancada')
            ], width={'size': 6, 'offset': 4}
            ),
            align='center'
        ),
        dbc.Row(
            [
                dbc.Col(width={'size': 3, 'offset': 1},
                        children=[
                        html.H3('Medições:'),
                        dcc.Dropdown(id='Arquivo',
                                        options=DROPDOWN_OPTIONS,
                                        multi=True,
                                        optionHeight=50
                                     ),
                            html.H3('Tipo de Gráfico:'),
                            dcc.RadioItems(
                                id='plot_domain',
                                options=[
                                    {'label': 'FFT', 'value': 'fft'},
                                    {'label': 'Tempo', 'value': 'time'}
                                ],
                                labelStyle={'display': 'block'},
                                value='time'
                        )
                        ]
                        ),
                dbc.Col(width={'size': 5, 'offset': 2},
                        children=[
                            html.H4('Tempo de Aquisição: 30s'),
                            html.H4(
                                'Frequência de Amostragem: 25600 Hz'),
                            html.H3('Spot:'),
                            dcc.RadioItems(
                                id='accel',
                                options=[
                                    {'label': 'Drive End Bearing',
                                        'value': 0},
                                    {'label': 'Non Drive End Bearing',
                                        'value': 1},
                                    {'label': 'Drive  End Motor',
                                        'value': 2},
                                    {'label': 'Fan End Motor',
                                        'value': 3}
                                ],
                                labelStyle={'display': 'block'},
                                value=0
                            )
                ]
                )
            ]
        ),
        dbc.Row(
            children=[
                dcc.Graph(id="plot"),
                dcc.Loading(dcc.Store(id='storage-data')),
                TraceUpdater(id='dynamic-updater', gdID='plot')
            ]
        ),
        dbc.Row(
            id='stats-card-row'
        )


    ]
)


# ----------------- App Callbacks -----------------------


@app.callback(
    ServersideOutput('storage-data', 'data'),
    Output('plot', 'figure'),
    Input('Arquivo', 'value'),
    Input('plot_domain', 'value'),
    Input('accel', 'value'),
    prevent_initial_call=True


)
def create_new_plot(dropdown_selection, plot_domain, acc):

    fig = create_figure(
        dropdown_selection, plot_domain=plot_domain, acc=acc)

    return fig, fig


@app.callback(
    Output('stats-card-row', 'children'),
    Input('Arquivo', 'value'),
    Input('plot_domain', 'value'),
    Input('accel', 'value'),
    prevent_initial_call=True
)
def create_stats_card(dropdown_selection, plot_domain, acc):

    card = dbc.Card(
        [
            dbc.CardHeader(html.H2('Estatísticas das medições')),
            dbc.CardBody(dbc.ListGroup(
                [
                    dbc.ListGroupItem(children=[
                        html.H3(stats["name"]),
                        html.H4(f'Média: {stats["mean"]}'),
                        html.H4(f'Valor RMS: {stats["rms"]}'),
                        html.H4(f'Skewness: {stats["skewness"]}')
                    ]
                    )
                    for stats in [get_stats(plot_domain, acc, file) for file in dropdown_selection]
                ],
                flush=True,
            ),
                style={"width": "100rem"},
            )])

    return card


@app.callback(
    Output("dynamic-updater", "updateData"),
    Input("plot", "relayoutData"),
    State("storage-data", "data"),
    prevent_initial_call=True,
    memoize=True,
)
def update_fig(relayoutdata: dict, fig: FigureResampler):
    if fig is not None:
        return fig.construct_update_data(relayoutdata)
    return no_update


# ----------- Run App --------------

if __name__ == '__main__':

    app.run_server(debug=True)
````

**Comment by jonasvdd** [comment]
Hi! 🤙 Can you provide a minimal (non-)working example with your behavior? 

Kind regards,
Jonas

**Comment by jonasvdd** [comment]
I suppose this issue is resolved within #261
